lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.html

```diff
@@ -101,7 +91,11 @@
                 [type]=property.inputType
                 class="adf-property-value"
                 title="{{property.label | translate }}"
-                [ngClass]="{ 'adf-textitem-clickable-value': !isEditable }"
+                [ngClass]="{
+                    'adf-property-value-editable': editable,
```

Comment:
I don't see this class in scss file for this component. Is not that missing?

Reply:
https://github.com/Alfresco/alfresco-ng2-components/blob/b00d12406ff70a52c49a04151f3da7cd98b83c4d/lib/core/src/lib/card-view/components/card-view/card-view.component.scss#L63